node/src/Worker.ts

```diff
@@ -63,7 +63,7 @@ export type WorkerSettings =
 	 * certificate is dynamically created.
 	 */
 	dtlsPrivateKeyFile?: string;
-
+	/* eslint-disable max-len */
```

Comment:
Please don't use this but `eslint-disable-next-line` instead and put it in the line above the problematic one.

Using `eslint-disable xxxx`  means that `xxx` rule gets disabled for the **whole file**.

Alternatively you can also use `eslint-disable xxxx` but then you MUST write `eslint-enable xxxx` after the problematic code.

Reply:
that was by mistake in latest commit it is fine. 
https://github.com/versatica/mediasoup/blob/becaef171cc512fe46523c2f2280a918d4601a59/node/src/Worker.ts#L66